include/cetl/_helpers.hpp

```diff
@@ -0,0 +1,94 @@
+/// @copyright
```

Comment:
I want to avoid widely entangled dependencies in CETL. Can we rename this to `_helper_enable_copy_move_construction.hpp` to avoid future abuse of this header as a dumping ground for syntactic sugar?